packages/site-kit/src/lib/components/docs/Main.svelte

```diff
@@ -200,20 +200,23 @@
 		font-size: 2.4rem;
 		color: var(--second);
 		margin-inline: 0;
-		margin-block: -5.5rem 1.6rem;
+		margin-block: 6.4rem 1.6rem;
 		padding-inline-start: 0;
 		background: transparent;
 		line-height: 1;
-		padding-block-start: 10rem;
+		padding-block-start: 0;
 		inset-block-start: 0;
 	}
 
-	.content :global(h4 > em) {
-		opacity: 0.7;
+	.content :global(h4::before) {
+		display: block;
+		content: ' ';
+		block-size: var(--nav-h);
+		margin-block-start: calc(-1 * var(--nav-h));
 	}
 
-	.content :global(h4 > .anchor) {
-		inset-block-start: 0.05em;
+	.content :global(h4 > em) {
```

Comment:
Github seems to be messing up the diff here, more readable diff is
![image](https://user-images.githubusercontent.com/50981692/147868171-a76288fa-b4b9-45f3-ab77-06bfdbb12952.png)